include/rocksdb/cloud/cloud_file_system.h

```diff
@@ -299,11 +299,12 @@ class CloudFileSystemOptions {
   bool resync_on_open;
 
   // Experimental option!
-  // This option only affects how resync_on_open works. If resync_on_open is true,
-  // and resync_manifest_on_open is true, besides fetching CLOUDMANFIEST from s3,
-  // we will fetch latest MANIFEST file as well.
-  // 
-  // This is a temporary option to help quickly rollback the change if something unexpected is wrong.
+  // This option only affects how resync_on_open works. If resync_on_open is
+  // true, and resync_manifest_on_open is true, besides fetching CLOUDMANFIEST
+  // from s3, we will fetch latest MANIFEST file as well.
+  //
+  // This is a temporary option to help quickly rollback the change if something
+  // unexpected is wrong.
   // TODO(wei): remove this option once we are confident about the change.
```

Comment:
Is the TODO still valid or can we get rid of it?

Reply:
@seckcoder I think we can get rid of this, right?

Reply:
@seckcoder, do you recall the conclusion here? Thanks!

Reply:
yeah, we should get rid of it. Will create a PR